pkg/operator/ceph/operator.go

```diff
@@ -199,6 +199,10 @@ func (o *Operator) startSystemDaemons(clusterSpec *cephv1.ClusterSpec) error {
 		return errors.Wrapf(err, "invalid csi params")
 	}
 
+	if err = csi.ValidateCSIVersion(o.context.Clientset, o.operatorNamespace, csi.CSIParam.CSIPluginImage, o.rookImage, o.securityAccount); err != nil {
```

Comment:
we don't need to pass CSIPluginImage, we can directly call it as `CSIParam.CSIPluginImage` in `spec.go` as this is defined as global variable in `spec.go`

Reply:
Good Point.